macros/src/lib.rs

```diff
@@ -492,7 +499,10 @@ pub fn pre_init(args: TokenStream, input: TokenStream) -> TokenStream {
 
 // Creates a random identifier
 fn random_ident() -> Ident {
-    let mut rng = rand::thread_rng();
+    let seed = SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_secs();
+    CALL_COUNT.fetch_add(1, Ordering::SeqCst);
```

Comment:
This returns the previous value. Why not use that instead of doing another atomic load?